amundsen_application/static/js/components/common/SearchBar/styles.scss

```diff
@@ -0,0 +1,81 @@
+@import 'variables';
+
+#search-bar {
+  .search-bar-form {
+    position: relative;
+
+    .search-button {
+      position: absolute;
+      height: 24px;
+      width: 24px;
+      top: 28px;
+      left: 24px;
+    }
+
+    .search-button-small {
+      position: absolute;
+      height: 24px;
+      width: 24px;
+      top: 6px;
+      left: 8px;
+    }
+
+    .clear-button {
+      position: absolute;
+      height: 24px;
+      width: 24px;
+      top: 2px;
+      right: 8px;
+    }
+
+    .search-bar-input {
+      border-radius: 4px;
+      border-color: $gray-light;
+      box-shadow: 0 3px 12px 0 rgba(17, 17, 31, 0.04);
+      width: 100%;
+
+      &.h2 {
+        height: 80px;
+        padding: 16px 24px 16px 64px;
+      }
+
+      &.h3 {
+        height: 36px;
+        padding: 6px 6px 6px 36px;
+      }
```

Comment:
It seems odd to rely on the `h2` and `h3` classes for resizing the search bar. It might be cleaner to put a `small` class on the root search bar element and then make rules like 

```
#search-bar {
  &.small {
    .search-button {
      ...
    }
    .search-bar-input {
      height ...
    }
  }
}
```

Reply:
The `h2` seemed to already be used to size the previous `SearchBar`. I initially added a `small` class, but then the inconsistency got confusing. Are you suggesting we using `small` anyway? Or perhaps update to `small` and `large` options for all the things that change?

Reply:
I think I would still prefer an explicit `.small` class over using the header classes. I see the `hX` classes are just shortcuts to predefined typography rules.

Reply:
Cool. Already handled in recent upload.